src/topology/continuous_function/algebra.lean

```diff
@@ -582,6 +582,36 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [semiring R] [topological_space R]
+  [topological_ring R] : has_continuous_mul C(α, R) :=
```

Comment:
Please move this to the section on monoids, state it for monoids rather than semirings, and `to_additive`-ize it.

Reply:
Done